ai_engine_sdk/client.py

```diff
@@ -273,14 +274,15 @@ async def get_messages(self) -> List[ApiBaseMessage]:
                 agent_json: dict = message['agent_json']
                 agent_json_type: str = agent_json['type'].upper()
                 if is_task_selection_message(message_type=agent_json_type):
-                    indexed_task_options: dict = get_indexed_task_options_from_raw_api_response(raw_api_response=message)
+                    indexed_task_options: dict = get_indexed_task_options_from_raw_api_response(
```

Comment:
here and so...

Reply:
Sorry for this formatting issue, I am looking into updating the code formatting in my editor. I have removed all of these formatting issues in my latest push.